Tests/DependencyInjection/Factory/Loader/ChainLoaderFactoryTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Tests\DependencyInjection\Factory\Loader;
+
+use Liip\ImagineBundle\DependencyInjection\Factory\Loader\ChainLoaderFactory;
+use Liip\ImagineBundle\Tests\DependencyInjection\Factory\FactoryTestCase;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\Processor;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * @covers \Liip\ImagineBundle\DependencyInjection\Factory\Loader\ChainLoaderFactory
+ */
+class ChainLoaderFactoryTest extends FactoryTestCase
+{
+    public function testImplementsLoaderFactoryInterface()
+    {
+        $rc = new \ReflectionClass('\Liip\ImagineBundle\DependencyInjection\Factory\Loader\ChainLoaderFactory');
```

Comment:
IDK if they are "needed" but for consistency with the other loader tests, I feel like we should include it for now. In the meantime, I have changed it to remove the reflection-based method used for the much more sensible use of PHPUnit's `isInstanceOf()`.